### ISSUE TITLE
Backport: Unable to launch catalog app by providing answers in yaml

### ISSUE THREAD

**Issue by alena1108** [rancher/rancher#19542]
Backport: https://github.com/rancher/rancher/issues/19511

**Comment by alena1108** [comment]
Can be tested with rc10

**Comment by davidnuzik** [comment]
Version: v2.2.2-rc10

Scenario A:
Confirmed that when I try to submit as form the UI will still prevent launching of the Authelia app since required fields are not filled out.

Scenario B:
Confirmed that I can now launch the app with the provided YAML (includes required LDAP Server URL, Base DN, Admin user, Admin user password)
Before UI was blocking due to it claiming that "LDAP Server URL" and "Base DN" were not provided even though they were.
```
authelia:
  enableHealtcheck: false
  config:
    domain: "example.com"
    logsLevel: "debug"
    session:
      secret: "changeme"
    authentication_backend: "ldap"
    ldap:
      url: "ldap://ldap.example.com"
      user: "cn=admin,dc=example,dc=com"
      password: "admin"
      base_dn: "dc=example,dc=com"
      additional_users_dn: ""
      users_filter: "cn={0}"
      additional_groups_dn: ""
      groups_filter: "(&(member={dn})(objectclass=group))"
      group_name_attribute: "cn"
      mail_attribute: "mail"
  ingress:
    enabled: true
    hosts:
      - "login.example.com"
```

Scenario C:
When I try to launch with YAML but do NOT provide required answers such as "LDAP Server URL" and "Base DN"  (null string provided) I can still launch the app - I should not be allowed to.
Expected behavior @loganhz ? (leodotcloud mentioned it in #19511 )
```
authelia:
  enableHealtcheck: false
  config:
    domain: "example.com"
    logsLevel: "debug"
    session:
      secret: "changeme"
    authentication_backend: "ldap"
    ldap:
      url: ""
      user: "cn=admin,dc=example,dc=com"
      password: "admin"
      base_dn: ""
      additional_users_dn: ""
      users_filter: "cn={0}"
      additional_groups_dn: ""
      groups_filter: "(&(member={dn})(objectclass=group))"
      group_name_attribute: "cn"
      mail_attribute: "mail"
  ingress:
    enabled: true
    hosts:
      - "login.example.com"
```

Scenario D:
When I try to launch with YAML but do NOT provide the ldap url and base_dn at all. I can still launch the app - I should not be allowed to. Expected behavior here as well @loganhz ? (similar to Scenario C and what leodotcloud mentioned in #19511 )
```
authelia:
  enableHealtcheck: false
  config:
    domain: "example.com"
    logsLevel: "debug"
    session:
      secret: "changeme"
    authentication_backend: "ldap"
    ldap:
      user: "cn=admin,dc=example,dc=com"
      password: "admin"
      additional_users_dn: ""
      users_filter: "cn={0}"
      additional_groups_dn: ""
      groups_filter: "(&(member={dn})(objectclass=group))"
      group_name_attribute: "cn"
      mail_attribute: "mail"
  ingress:
    enabled: true
    hosts:
      - "login.example.com"
```
^^^ ldap url / base_dn missing

**Comment by davidnuzik** [comment]
Closing issue; based on discussion in Slack this is a temporary measure in v2.2.2 -- this at least allows us to launch the app at all in v2.2.2 with YAML so the UI doesn't block us for "fields" it thinks were not filled out. We are planning to fully fix in v2.3 so that hopefully scenario C and D also work (see my above testing notes)
